pulsetranslator/pulsetranslator.py

```diff
@@ -25,6 +25,9 @@
 
 
 class PulseBuildbotTranslator(object):
+    # Available buildbot results
+    # http://hg.mozilla.org/build/buildbot/file/08b7c51d2962/master/buildbot/status/builder.py#l25
+    results = ['success', 'warnings', 'failure', 'skipped', 'exception', 'retry']
```

Comment:
Depending on this will break as soon as builds move into TaskCluster, but that's probably unavoidable for now.